models/brac/views/hhview_visits.sql

```diff
@@ -0,0 +1,23 @@
+SELECT 
+	upr.reported AS reported,
+	upr.reported_by_parent,
+	cvp.parent_uuid AS household_id
+FROM 
+	{{ ref("useview_patient_record") }} AS upr
+INNER JOIN {{ ref("contactview_person") }} cvp ON (upr.patient_id=cvp.patient_id)
+UNION 
```

Comment:
UNIONS are quite expensive. Could you liaise with Jonathan to find would why we are using UNION rather than UION ALL?

Reply:
All 3 DB objects used in the union ([useview_place_record](https://github.com/medic/config-brac/blob/master/postgres/materialized_views/useview_place_record.sql), [useview_patient_record](https://github.com/medic/config-brac/blob/master/postgres/materialized_views/useview_patient_record.sql), contactview_metadata) all select directly from couchdb table.

So UNION is necessary to avoid duplicates.

Reply:
Unless you can suggest a more perfomant version of this.